programmers/카펫.js

```diff
@@ -0,0 +1,14 @@
+function solution(brown, yellow) {
+    
+    for (let i = 0; i <= yellow; i += 1) {
+        if (yellow % i === 0) {
+            let h = i;
+            let w = yellow / h;
+            
+            if (brown === w * 2 + h * 2 + 4) {
+                return [w + 2, h + 2]
+            }
+        }
```

Comment:
```suggestion
        const h = i;
        const w = yellow / h;
        if (yellow % i === 0 && brown === w * 2 + h * 2 + 4) {
            return [w + 2, h + 2]
        }
```
꼭 const로 써 주시고, 위에 처럼 순서만 바꿔주시면 조건문을 한 번만 사용할 수도 있어요.